scripts/utils/createJestConfig.js

```diff
@@ -21,8 +21,8 @@ module.exports = (resolve, rootDir) => {
   const config = {
     moduleFileExtensions: ['jsx', 'js', 'json'],
     moduleNameMapper: {
-      '^[./a-zA-Z0-9$_-]+\\.(jpg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm)$': resolve('config/jest/FileStub.js'),
-      '^[./a-zA-Z0-9$_-]+\\.css$': resolve('config/jest/CSSStub.js')
+      '^.+\\.(jpg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm)$': resolve('config/jest/FileStub.js'),
```

Comment:
If you are going to ignore everything before the end, you can just omit `^.+` altogether and just write `\\.css$`